syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
Hi Heath, I pulled a copy of you profile, opened it, and temporarily forced functions/cmdlets to bright red with `:hi Function guifg=#FF0000` and here's what I get.  Does this look right?

![capture-cmdlets](https://cloud.githubusercontent.com/assets/427467/18973740/695e2918-86e2-11e6-83b3-3bf21c6f0308.png)

Cmdlets/functions are now recognised as `Function` instead of `Statement`.  Maybe your color scheme has different colors for the two?